merge_rgba/__init__.py

```diff
@@ -109,19 +110,22 @@ def merge_rgba_tool(sources, outtif, bounds=None, res=None, precision=7,
 
                 # Alternative, custom get_window using rounding
                 window_start = rowcol(
-                    src.transform, left, top, op=round, precision=precision)
+                    src.transform, left, top, op=round, precision=precision
+                )
                 window_stop = rowcol(
-                    src.transform, right, bottom, op=round, precision=precision)
+                    src.transform, right, bottom, op=round, precision=precision
+                )
                 src_window = tuple(zip(window_start, window_stop))
 
                 temp = np.zeros(dst_shape, dtype=dtype)
-                temp = src.read(out=temp, window=src_window,
-                                boundless=True, masked=False)
+                temp = src.read(
+                    out=temp, window=src_window, boundless=True, masked=False
+                )
 
                 # pixels without data yet are available to write
                 write_region = np.logical_and(
-                    (dstarr[3] == 0),  # 0 is nodata
-                    (temp[3] != 0))
+                    (dstarr[3] == 0), (temp[3] != 0)  # 0 is nodata
```

Comment:
this black change is a little hard to swallow -  rearranging the code here could change the interpretation of inline comments.

Reply:
This behavior has been noted as a bug: https://github.com/ambv/black/issues/282